### ISSUE TITLE
Try /subreddits/default API to get 100 subreddits at a time

### ISSUE THREAD

**Issue by stephengtuggy** [michaelachrisco/subreddit-recommender#32]
https://www.reddit.com/dev/api#GET_subreddits_default

**Comment by stephengtuggy** [comment]
Maybe use the optional sr_detail parameter to get details of each subreddit in the same call.

**Comment by michaelachrisco** [comment]
Good idea. We can roll some optional parameters into each of the REST calls. Retrieving only 10/100/1000/ etc with pagination would be useful for any application consuming the data.

**Comment by stephengtuggy** [comment]
I'm talking about what URL(s) subreddit-recommender itself should use to retrieve its source data from reddit.com. Are we talking about the same thing?

**Comment by stephengtuggy** [comment]
Reddit.com strongly prefers batch operations over retrieving single records at a time. See https://github.com/reddit/reddit/wiki/API , 5th bullet under "Rules" heading.
